libs/application/templates/social-insurance-administration/medical-and-rehabilitation-payments/src/forms/MedicalAndRehabilitationPaymentsForm/generalInformationSection/questionsSubSection.ts

```diff
@@ -103,10 +103,9 @@ export const questionsSubSection = buildSubSection({
           width: 'half',
           loadingError: coreErrorMessages.failedDataProvider,
           loadOptions: async ({ apolloClient }) => {
-            const { data } =
-              await apolloClient.query<SiaEducationalInstitutionsQuery>({
```

Comment:
Here we are losing operation generated type safety.

Could keep a codegen setup for operations while still using api/schema base types. The portals do this a lot:

https://github.com/island-is/island.is/blob/a3f17e013a1dd0932ece2b5fb7307c4136b3575e/libs/portals/my-pages/education/codegen.yml